data/hai/culture hai.txt

```diff
@@ -0,0 +1,208 @@
+			choice
+				`	"Your music comes off a production line?"`
+					goto production
+				`	"Do you not have musicians trying to make it on their own?"` 
+			`	"Oh for sure, but mostly just to get picked up by one of these companies."`
+			label production
+			`	The bartender chuckles. "We've been doing pop music for longer than your people have had fire, it's about the experience of being a fan of a group as it rises to fame and stardom. A dedicated Hai can go back and find a song for almost any possible life experience in almost any possible genre. It's the journey, and the personalities, that are what it's all about."`
+			choice
+				`	"How strange."`
+				`	"How interesting."`
+			`	"Anyway, I gather you were you after some food?"`
+			label wrapup
+			`	The bartender tips his head in your direction and takes your order. "No problem, it'll be right up."`
+			`	It's a noisy meal.`
+				decline
+
+#This uses the flag from receiving the Hai translator.
+mission "ZZCU The Meme Wars 1"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Stepping into the spaceport you are beset by a riot of color. Every blank surface that's not usually paid advertising space seems to be festooned with dangling holographic projectors, ribbons, banners, and even actual paper fliers and posters, apparently taped to whatever surface will bear them. At first you wonder if there's some sort of festival going on, but as you watch a young Hai wearing a hood slaps up another holo projector which immediately projects the image of an anthro root vegetable with a sword hacking at some kind of impressionist concept of a Hai elder made up of iridescent triangles. The young Hai then proceeds to tear down a banner and slap some goo on a massive poster which begins dissolving before a pair of security guards, wearing some very obviously non-standard additions to their uniforms, chase him back out of the spaceport.`
+			branch translator
+				has "language: Wanderer"
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance, before proceeding to act out something that makes precisely no sense to you.`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost sir but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`
```

Comment:
I hate to say "cringe" in 2021, but **this is cringe.**
Aside from the usual wonky pacing, DO NOT USE THE WORD "MEME" IN THE GAME PLEASE I BEG YOU.

*cough* sorry i lost my cool

Reply:
Meme might be popularised right now, but it's a legitimate term to describe the phenomenon. In fact there is no equivalent term in english which can be substituted and this is a rare case where society has popularised the correct scientific use of a term.